### PR TITLE
Merge tobrun 0.12

### PR DESCRIPTION
**Pull request by vberthet**
Merge change from https://github.com/tobrun/flutter-mapbox-gl/tree/0.12.0
It also include iso build fix from https://github.com/m0nac0/flutter-maplibre-gl/pull/7

### CODE REVIEW

**Comment by m0nac0** [comment]
@vberthet Thank you for contributing! I've started to review the changes, but since they are pretty extensive I'll need a bit more time to work through them.

File: maplibre_gl_platform_interface/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+## 0.12.0, April 12, 2020
```
Comment (m0nac0) [inline maplibre_gl_platform_interface/CHANGELOG.md]: We may want to put a note here about this being the original changelog (or the changes we have cherry-picked, respectively)

File: maplibre_gl_web/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,13 @@
+## 0.12.0, April 12, 2020
```
Comment (m0nac0) [inline maplibre_gl_web/CHANGELOG.md]: Same as for the other changelog

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,28 @@
 ## Below is the original changelog of the tobrun/flutter-mapbox-gl project, before the fork.
 
+## 0.12.0, April 12, 2020
```
Comment (m0nac0) [inline CHANGELOG.md]: I'd like to move this section of changelogs above the "## Below is the original changelog..." heading and put it under a separate heading like:
 "## Changes were cherry-picked/ported from the following releases of the tobrun/flutter-mapbox-gl project into this project"

File: pubspec.yaml [hunk 4]
```diff
@@ -1,6 +1,6 @@
 name: maplibre_gl
 description: A Flutter plugin for integrating Maplibre Maps inside a Flutter application on Android, iOS and web platforms.
-version: 0.10.0
+version: 0.12.0
```
Comment (m0nac0) [inline pubspec.yaml:3]: See #13

**Comment by m0nac0** [comment]
@vberthet Sorry for taking so awfully long with this and again thanks for opening the PR. I left some comments and have worked through most of the changes, just some of the iOS stuff left to work through...

**Comment by m0nac0** [comment]
@vberthet I've solved the merge conflicts locally, if you'd like me to push the merge commit you'd need to edit the settings for the PR to allow me that. (https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork)

If you'd rather do it yourself that is also fine, though!

**Comment by m0nac0** [comment]
@vberthet I finally finished the review! As soon as the merge conflict is resolved, I'd love to merge this!

I could just address the remaining comments regarding the changelogs after this PR is merged.

**Comment by vberthet** [comment]
@m0nac0 I've updated the CHANGELOG files

**Review by m0nac0 — APPROVED** [review]
Great work, thank you for the updates!

**Pull request merged**